src/cmds/openapi/index.ts

```diff
@@ -222,11 +223,8 @@ export default class OpenAPICommand extends Command {
     const options: RequestInit = {
       headers: cleanHeaders(
```

Comment:
I refactored the signature of our `cleanHeaders` function so the second parameter is a `version` which we use to construct our `x-readme-version` request header. You'll see a lot of refactors related to that in this PR.